<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainArmouryWindow</class>
 <widget class="QMainWindow" name="MainArmouryWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>631</width>
    <height>545</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <property name="windowIcon">
   <iconset resource="../../../libwar/resources/images.qrc">
    <normaloff>:/icons/sword-preview.png</normaloff>:/icons/sword-preview.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="WarPageViewer" name="pageViewer"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>631</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuF_ile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionOpenFile"/>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuF_ile"/>
  </widget>
  <action name="actionOpenFile">
   <property name="icon">
    <iconset resource="../../../libwar/resources/images.qrc">
     <normaloff>:/icons/document-open.png</normaloff>:/icons/document-open.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Open File</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="../../../libwar/resources/images.qrc">
     <normaloff>:/icons/application-exit.png</normaloff>:/icons/application-exit.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>WarPageViewer</class>
   <extends>QTabWidget</extends>
   <header location="global">widgets/WarPageViewer.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../libwar/resources/images.qrc"/>
 </resources>
 <connections/>
</ui>
